### PR TITLE
Singularity does not throw items

### PR DESCRIPTION
**Pull request by Robertha89**
While I loved the concept, i hate the fact that it's the most resource intensive part of the singularity, just 2 rounds ago, a supermatter singularity was born, and then it ate at 40 chickens, before throwing 10-20 items at them. So yes, remove this to make singularity less resource intensive.

### CODE REVIEW

File: code/game/objects/items.dm [hunk 1]
```diff
@@ -499,10 +499,7 @@
 			A.blood_overlay = image(I)
 
 /obj/item/singularity_pull(S, current_size)
-	spawn(0) //this is needed or multiple items will be thrown sequentially and not simultaneously
-		if(current_size >= STAGE_FOUR)
-			throw_at(S,14,3)
-		else ..()
+	return
```
Comment (tigercat2000) [inline code/game/objects/items.dm]: this needs to be either `. = ..()` or `return ..()`, pick one

**Comment by Robertha89** [comment]
I believe this should fix it, please respond if not.

**Comment by tigercat2000** [comment]
that definately doesn't fix it, now you have items completely ignoring their parent /singularity_pull

**Comment by tigercat2000** [comment]
just delete the entire `item/singularity_pull`

**Comment by orelbon** [comment]
so the singulo cant suck in items anymore?

**Comment by Fox-McCloud** [comment]
This isn't the majority of the singularity lag, at all.

If you profile it, the majority of the lag comes from plain old Del(); with the amount of stuff being destroyed, even qdel() starts lugging things down.

This won't make the singularity much less laggy, at all.

Also, you made a super singularity--that's going to be even more exceptionally laggy.

**Comment by Robertha89** [comment]
true, the main reason is del(), but it should still provide an increased performance, as watching 7 different chickens getting hit by a dozen things, tends to freeze up the game somewhat.

**Comment by Fox-McCloud** [comment]
I almost guarantee you it was the 40 chickens getting hard deleted than the chickens getting hit by items that incurred that much lag.

**Comment by Robertha89** [comment]
No, I was observing it all, and it gibbed 3 chickens at a time before throwing a dozen items at the closest ones.

**Comment by Fox-McCloud** [comment]
If you can prove your point, definitively then and the performance is markedly better, then I'd be willing to concede my point, but until then...

**Comment by Fox-McCloud** [comment]
https://gyazo.com/1fdc25f2a8cdf4dadd473d2db9210312
https://gyazo.com/68010e57fb2765a26d8076370ad3ddfd

In the grand scheme of things, it barely registers.

**Comment by Fox-McCloud** [comment]
Going to close this--the amount of impact it has, performance wise, is small--removing it for performance reasons really doesn't make much sense.

**Pull request closed without merging**